README.md

```diff
@@ -21,7 +21,7 @@ If you use AMD, UMD, Commonjs or just need the script in browser then install vi
 In **Node**
 
 ```js
-const frictionlessCkanMapper = require('frictionless-ckan-mapper')
+const frictionlessCkanMapper = require('frictionless-ckan-mapper-js')
```

Comment:
i would suggest:

```
const mapper = require('frictionless-ckan-mapper-js')
```

Reply:
Also lib name would normally be without the `-js` no?

Reply:
> Also lib name would normally be without the `-js` no?

Yea I see, I've intentionally made so, just to not confuse with the original one. I'm totally fine to name it without `-js` if you want to me make that change.